input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
+AU Core is designed to ensure that core digital health and administrative data can be recorded, updated, searched, and retrieved efficiently, promoting interoperability and adoption across healthcare systems. By aligning with key FHIR implementation guides such [International Patient Access (IPA) 1.0.0)](https://hl7.org/fhir/uv/ipa/STU1/), [International Patient Summary Implementation Guide (IPS) 1.1.0](https://hl7.org/fhir/uv/ips/STU1.1/) and [US Core Implementation Guide 7.0.0](https://hl7.org/fhir/us/core/2024Jan/), AU Core ensures consistency with both international and national requirements.
```

Comment:
Replace the text up to the heading `#### Profile comparison` with:

```
AU Core aligns to, and leverages, international standards and other national standards, in particular:
- [International Patient Access 1.0.0](https://hl7.org/fhir/uv/ipa/STU1/)
- [International Patient Summary Implementation Guide 1.1.0](https://hl7.org/fhir/uv/ips/STU1.1/)
- [US Core Implementation Guide 7.0.0](https://hl7.org/fhir/us/core/2024Jan/)

Relationships between AU Core, AUCDI, and the above key implementation guides is described in [Relationship with other IGs](relationship.html). 

Corresponding profiles, conformance requirements, and capability statements included in the key FHIR implementation guides were reviewed and considered during AU Core's development to ensure alignment, and to facilitate adoption of, this standard.

The below comparison evaluates AU Core conformance requirements with the key implementation guides identifying where compliance with AU Core satisfies the expectations established by the referenced implementation guide. 

At this time only profile comparison is available. Future versions of AU Core will compare capability statements.
```

Reply:
Done